### PR TITLE
Feature(openIdConnect) : optimize openIdConnect query client

### PR DESCRIPTION
**Pull request by 15168440402**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Review by zhenlei520 — APPROVED** [review]
LGTM

File: src/Contrib/Authentication/OpenIdConnect/Tests/Masa.Contrib.Authentication.OpenIdConnect.EFCore.Tests/SyncCacheTest.cs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
+using Masa.BuildingBlocks.Authentication.OpenIdConnect.Domain.Entities;
```
Comment (zhenlei520) [inline src/Contrib/Authentication/OpenIdConnect/Tests/Masa.Contrib.Authentication.OpenIdConnect.EFCore.Tests/SyncCacheTest.cs:4]: use global using

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

**Comment by doddgu** [comment]
[11.1% Coverage](https://sonarcloud.io/component_measures?id=masastack_MASA.Framework&pullRequest=383&metric=new_coverage&view=list)

**Review by zhenlei520 — APPROVED** [review]
LGTM @doddgu

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**